cinema/views.py

```diff
@@ -48,6 +87,42 @@ class MovieSessionViewSet(viewsets.ModelViewSet):
     queryset = MovieSession.objects.all()
     serializer_class = MovieSessionSerializer
 
+    @staticmethod
+    def _param_to_date(query_string):
+        return datetime.datetime.strptime(query_string, "%Y-%m-%d")
```

Comment:
this will work if the format is valid but keep in mind that in reality we would need to validate the input which can be anything (raise 400 if invalid)
and by the way it's not called anywhere :)